lab_4/lab_4.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+using namespace std;
+
+void power(int digit) {

```

Comment:
Here and below: your functions only should perfom some calculations and return the result. They shouldn't ask user for anything, use output messages and so on.